https://issues.redhat.com/browse/ACM-5763 (#5166)
open-cluster-management/rhacm-docs: release_notes/known_issues_console.adoc

Reviewer: Make sure to remove any future tense.

After provisioning a hosted control plane cluster, you might not be able to scroll horizontally in the cluster overview of the {product-title-short} console if the `ClusterVersionUpgradeable` parameter is too long. You cannot view the hidden data as a result.

To work around the issue, zoom out by using your browser zoom controls, increase your {product-title-short} console window size, or copy and paste the text to a different location.

> [#editapplication set expand feature repeats]

Reviewer 2: anchors require a dash 
```suggestion
[#editapplicationset-expand-feature-repeats]
```

Reviewer 3: I am not sure we typically write **To work around** or **As a work around**?

Let's eliminate "might not be able" here. We need to think of the most concise way (avoiding extra words) to phrase things.

Example: Active, concise

After you provision a hosted control plane cluster, you might lose part of the `ClusterVersionUpgradeable` parameter when you scroll horizontally in the cluster overview of the {product-title-short} console. 

To resolve this issue, zoom out by using your browser zoom controls, increase your console window size, or copy and paste the text to a different location.

Reviewer 3: mentioning product one time should suffice here in a short entry.

> == EditApplicationSet expand feature repeats

Reviewer 2: We typically italicize resource names in the section title. This is because the resource names are monospaced according to the IBM Style Guide. I recommend that you italicize "EditApplicationSet"

```suggestion
== _EditApplicationSet_ expand feature repeats
```

> When making edits to your ApplicationSet on placement, you may encounter the recurrence of "expand to enter expression" when you attempt to enter your cluster selector. You will be able to enter your cluster selection despite this.

Reviewer 2: What is `ApplicationSet` on placement? ApplicationSet and Placement are both resource types. I read through the issue and saw some of the pictures 

```suggestion
When you add multiple label expressions or attempt to enter your cluster selector for your `ApplicationSet`, you might receive the following message repeatedly,  "Expand to enter expression". You can enter your cluster selection despite this issue.
```

Reviewer 3: That is also a good example of active voice @dockerymick , thanks for showing this.
